.github/CODEOWNERS

```diff
@@ -71,8 +71,8 @@ exporter/mezmoexporter/                                             @open-teleme
 exporter/opencensusexporter/                                        @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
 exporter/opensearchexporter/                                        @open-telemetry/collector-contrib-approvers @Aneurysm9 @MitchellGale @MaxKsyunz @YANG-DB
 exporter/otelarrowexporter/                                         @open-telemetry/collector-contrib-approvers @jmacd @moh-osman3 @lquerel
-exporter/prometheusexporter/                                        @open-telemetry/collector-contrib-approvers @Aneurysm9
-exporter/prometheusremotewriteexporter/                             @open-telemetry/collector-contrib-approvers @Aneurysm9 @rapphil
+exporter/prometheusexporter/                                        @open-telemetry/collector-contrib-approvers @Aneurysm9 @dashpole
```

Comment:
I believe you need to update https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusexporter/metadata.yaml#L11 too, same for other components

Reply:
I believe I did that in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34795/commits/f1b2698e0bc8861e8f5a26fe72cc68b08dbf9e58